README.md

```diff
@@ -580,6 +582,61 @@ The meta directive does not have any content and it accepts the following option
 * `points-to-pass`: module points to pass
 * `introduction`: module introduction as an HTML string
 
+Example module index.rst file:
+
+```rst
+Module 1 - Introduction
+=======================
+
+.. toctree::
+
+  introduction
+  topic
+
+.. aplusmeta::
+  :open-time: 2020-09-01 10:00
+  :close-time: 2020-09-30 14:00
+```
+
+Alternatively, one can also define these options in the conf.py file of the
+course in the following way.
+
+1. Include the aplusmeta directive in the `extensions` part of conf.py. Its
+   extension name is 'meta'.
+
+```python
+    extensions = [
+        ...
+        'meta',
+        ...
+    ]
+```
+
+2. Add the `aplusmeta_substitutions` variable in the conf.py file.
+
+```python
+    aplusmeta_substitutions = {
+        'open01': '2020-09-10 10:01'
+    }
+```
+
+This variable is a dictionary where keys are strings and values are dates
+in the usual format (see above). In the example above, you have defined a
+shortcut text "open01" for date "2020-09-10 10:01".
+
+3. Use the shortcut texts in the aplusmeta directive in a module index.rst:
+
+```rst
+    .. aplusmeta::
+      :open-time: open01
+      :close-time: 2019-09-20 12:00
+      :late-time: 2019-12-20 09:00
+```
+
+4. When the course is compiled, the aplusmeta directive looks for substitution
+strings in the `aplusmeta_substitutions` dictionary. The substitutions can be
+named freely as long as they are not RST markup (e.g. '|open01|' will not
```

Comment:
Would RST substitutions work to implement this feature? Would they be easier? Could they be defined in the `index.rst` for example (I don't think so, but would be cool)?

Reply:
RST does not support normal substitutions in the directive options and that is the reason for doing a custom implementation in this PR.

Reply:
Yes, exactly as Mankro said.